ADR/adr-0008-environment-provisioning.md

```diff
@@ -0,0 +1,464 @@
+# ADR-0008 Environment Provisioning
+
+Date: 2022-12-14
+
+## Status
+
+Accepted
+
+## Approvers
+
+* Alexey Kazakhov
+* Ann Marie Fred
+* Elson Yuen
+* Gorkem Ercan
+* Jonathan West
+* Krunoslav Pavic
+* Scott Hebert
+* Shoubhik Bose
+
+## Authors
+
+* Gal Ben Haim
+* Ralph Bean
+
+## Context
+
+In our old KCP architecture, we had [a
+design](https://docs.google.com/document/d/1WKd1FVHAxaNQKCIzIW-vUQRgsoOP9T-8rYozAMDpYc0/edit#) for
+provisioning a new deployment target in support of new [Environments].  This design was to be
+implemented in [GITOPSRVCE-228](https://issues.redhat.com/browse/GITOPSRVCE-228) by an environment
+controller that would create and manage sub-workspaces of the user’s main AppStudio workspace, and
+that would provide a serviceaccount to Argo in order to deploy the user’s application to those
+sub-workspaces. Now, without KCP, we need a new design.
+
+The [Environment] CR serves two purposes:
```

Comment:
Is `Environment` supposed to be a link ?

Reply:
It is. The link references are at the _very_ end of the file.